server/api/api/api.py

```diff
@@ -192,6 +193,11 @@
     tags=["artifacts"],
     dependencies=[Depends(deps.authenticate_request)],
 )
+api_v2_router.include_router(
+    alert_template.router,
+    tags=["alert-templates"],
+    dependencies=[Depends(deps.authenticate_request)],
+)
```

Comment:
Why does this need to be `v2`? 
The logic is that we still add endpoints to `v1` as long as we don't introduce a new version to an existing endpoint. Since these are brand-new endpoints, there's no real reason to use `v2` here.

Reply:
you defined it to be v2: https://iguazio.atlassian.net/wiki/spaces/ARC/pages/1867801/SDK+and+alert+usage

Reply:
I wonder what my 6-month ago self was thinking 😰